tests/integration/conftest.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+from typing import Generator
+from unittest.mock import patch
+from flask import Flask
+from flask.testing import FlaskClient
+import pytest
+from sqlalchemy.orm import Session
+
+from trailblazer.server.app import app
+from trailblazer.store.database import get_session
+from trailblazer.store.models import Analysis
+from trailblazer.store.store import Store
+from trailblazer.constants import PRIORITY_OPTIONS, TYPES, TrailblazerStatus, WorkflowManager
+
+
+@pytest.fixture
+def flask_app(store: Store):
+    yield app
+
+
+@pytest.fixture
+def client(flask_app: Flask) -> Generator[FlaskClient, None, None]:
+    # Bypass authentication
```

Comment:
It would be nice to include authentication in the integration tests for our API but I did not feel like dealing with it now. Can be added in later or tested separately.